### PR TITLE
fix(preset): refactor value preset #60

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #60 

## Summary:

_Please summarize the scope of the changes you have submitted, what the intent of the work is and anything that describes the before/after state of the project._

## Type of change:

Please delete options that are not relevant.

- [ ] New capability
- [x] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**
-- Auro Design System Team

### CODE REVIEW

File: demo/apiExamples.md [hunk 1]
```diff
@@ -76,7 +76,48 @@ A baseline `<auro-select>` using `<auro-menu>` and `<auro-menuoption>` elements.
 
 ### Property Examples
 
-#### <a name="placeholder"></a>`placeholder`
+#### value
+
+Use the `value` property to define a preset value on the `auro-select` element. The `value` of `auro-select` must match the `value` defined of a single `auro-menuoption`. Upon initially rendering the component, if the `value` does not match an `auro-menuoption`, the `value` of `auro-select` will be set to `undefined`.
+
+While `auro-menu` supports presetting the selection of an `auro-menuoption` by applying the `selected` attribute on the `auro-menuoption`, that should never be done when directly setting the `value` property on `auro-select`.
```
Comment (blackfalcon) [inline demo/apiExamples.md:83]: Why can we not have the pre-selected value also be supported by the `selected` attribute?

**Review by blackfalcon — CHANGES_REQUESTED** [review]

**Comment by trafton** [comment]
Checking on the status of this as it is blocking a story for the shopping team https://itsals.visualstudio.com/E_Sell_Shop/_workitems/edit/730474

**Comment by blackfalcon** [comment]
It's still a draft PR and I know @jason-capsule42 is working on combobox features. @jason-capsule42 what's the plan to wrap this one up?

**Comment by jason-capsule42** [comment]
@blackfalcon There is a dependency error in the package-lock.json that needs to get resolved for the actions to complete correctly. I'm not sure how to fix this one but if you can figure it out please let me know what the solution was.

File: demo/apiExamples.md (hunk 1 above)
Comment (blackfalcon) [inline demo/apiExamples.md:83]:
```suggestion
To pre-set the value of auro-select on load, use the `value` property. The `selected` attribute on auro-menuoption is designed to illustrate state in the DOM.
```

Let's rephrase this to be direct with intent versus potentially creating convulsion about the different API options.

**Review by blackfalcon — CHANGES_REQUESTED** [review]
My only feedback is on the docs description. Once that update is resolved, this is good to ship.
